modules/protocol/inspircd3.cpp

```diff
@@ -1105,6 +1114,9 @@ struct IRCDMessageCapab : Message::Capab
 				}
 				else if (module.equals_cs("m_topiclock.so"))
 					Servers::Capab.insert("TOPICLOCK");
+				else if (module.find("m_cban.so") == 0)
+					if (module.length() > 10 && module.substr(10) == "glob")
```

Comment:
This is probably fine-

```suggestion
				else if (module.equals_cs("m_cban.so=glob"))
```

Reply:
Heh, this is what happens when you follow the existing logic a little too closely :)